cmake/FindHepMC.cmake

```diff
@@ -0,0 +1,55 @@
+#Author: Kamil Sobol
```

Comment:
Is this file used? why would we need a dependency to HepMC? at least, it should be optional.